config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   get '/', to: 'welcome#index'
+  resources :movies, only: [:show]
+  resources :viewing_party, only: [:new]
```

Comment:
We might need to change the name of this resource to 'view_party'

Reply:
Sure! I can change it.